draft-ietf-jsonpath-base.md

```diff
@@ -1234,30 +1222,39 @@ The `filter-selector` works with arrays and objects exclusively. Its result is a
 A relative path, beginning with `@`, refers to the current array element or member value as the
 filter selector iterates over the array or object.
 
-When a path resulting in an empty nodelist appears on either side of a comparison, the result of the comparison is
+A singular path by itself in a Boolean context is an existence test which yields true if the path selects a node and yields false if the path does not select a node.
+To be more specific about the actual value of a node selected by a path, explicit comparisons are necessary. This existence test — as an exception to the general rule — also works with nodes with structured values.
+
+When a path resulting in an empty nodelist appears on either side of a comparison, the comparison yields
 true if and only if the comparison operator is `==`, `>=` or `<=` and the other side of the comparison is also a path
```

Comment:
So != is always false, while == can be true?
Doesn't look right.

Reply:
I tried fixing this in https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/224.